src/rebar_port_compiler.erl

```diff
@@ -568,9 +568,21 @@ erl_interface_dir(Subdir) ->
     end.
 
 default_env() ->
+    Arch = os:getenv("REBAR_TARGET_ARCH"),
+    Vsn = os:getenv("REBAR_TARGET_ARCH_VSN"),
     [
-     {"CC" , "cc"},
-     {"CXX", "c++"},
+     {"CC" , get_tool(Arch,Vsn,"gcc","cc")},
+     {"CXX", get_tool(Arch,Vsn,"g++","c++")},
+     {"AR" , get_tool(Arch,"ar","ar")},
+     {"AS" , get_tool(Arch,"as","as")},
+     {"CPP" , get_tool(Arch,Vsn,"cpp","cpp")},
+     {"LD" , get_tool(Arch,"ld","ld")},
+     {"RANLIB" , get_tool(Arch,Vsn,"ranlib","ranlib")},
+     {"STRIP" , get_tool(Arch,"strip","strip")},
+     {"NM" , get_tool(Arch,"nm","nm")},
+     {"OBJCOPY" , get_tool(Arch,"objcopy","objcopy")},
+     {"OBJDUMP" , get_tool(Arch,"objdump","objdump")},
```

Comment:
Here and in the rebar_utils diff, for consistency, `,` should be followed by a space. Also some of the leading spaces alignment above looks inconsistent. As alignment like that leads to extra whitespace changes whenever the list is modified, it's preferable to not align at all.